easybuild/easyconfigs/x/Xmipp/Xmipp-3.19.04-Apollo-foss-2019a-Python-2.7.15.eb

```diff
@@ -0,0 +1,93 @@
+name = 'Xmipp'
+version = '3.19.04-Apollo'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/I2PC/scipion/wiki'
+description = """Scipion is an image processing framework to obtain 3D
+models of macromolecular complexes using Electron Microscopy (3DEM). It
+integrates several software packages and presents an unified interface
+for both biologists and developers. Scipion allows to execute workflows
+combining different software tools, while taking care of formats and
+conversions. Additionally, all steps are tracked and can be reproduced
+later on. 
+"""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+sources = [
+    {
+        'source_urls': ['https://github.com/I2PC/xmipp/archive/'],
+        'download_filename': 'v%(version)s.tar.gz',
+        'filename': '%(namelower)s-v%(version)s.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/I2PC/xmippCore/archive/'],
+        'download_filename': 'v%(version)s.tar.gz',
+        'filename': 'xmippCore-v%(version)s.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/I2PC/xmippViz/archive/'],
+        'download_filename': 'v%(version)s.tar.gz',
+        'filename': 'xmippViz-v%(version)s.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/DStrelak/cuFFTAdvisor/archive/'],
+        'download_filename': '14523b8ca7e2f5474feae49eb5ac8ca1fa25f5cd.tar.gz',
+        'filename': 'cuFFTAdvisor-20191010.tar.gz',
+    },
+]
+patches = [
+    '%(name)s-%(version)s_fix_scons_Environent_calls.patch',
+    '%(name)s-%(version)s_use_CXXFLAGS_from_EB.patch',
+    '%(name)s-%(version)s_add_missing_pthread_to_XmippParallel.patch',
+    '%(name)s-%(version)s_add_missing_pthread_to_XmippInterfaceCuda.patch',
+    '%(name)s-%(version)s_dont_add_useless_libs.patch',
+    '%(name)s-%(version)s_correct_order_of_cu_includes.patch',
+    'XmippCore-%(version)s_fix_scons_Environent_calls.patch',
+    'XmippCore-%(version)s_use_CXXFLAGS_from_EB.patch',
+    'XmippViz-%(version)s_fix_scons_Environent_calls.patch',
+    'XmippViz-%(version)s_use_CXXFLAGS_from_EB.patch',
+]
+checksums = [
+    '1fcad53aacafca5c1ecf4350b83679178c40e8fd1984282a9e2359e2ae5b84e0',  # xmipp-v3.19.04-Apollo.tar.gz
+    '0f540db0e8e264443974ba761d447b7c30bfdd6f63258a010e4875edcf658487',  # xmippCore-v3.19.04-Apollo.tar.gz
+    '49a4180f3f9a07208b78715a3a6a47d02d0537008d4c25d9812c1574f14f2676',  # xmippViz-v3.19.04-Apollo.tar.gz
+    '3c0de8b3119e9e6db783841254a6e2702dadf6df4e96baaee0b5c95216481621',  # cuFFTAdvisor-20191010.tar.gz
```

Comment:
@akesandgren Checksum wasn't updated after downloading tarball rather than using `git_config`?

Should be `2d97cc1c9c955fa5bee44eb232d49f25f2c8ad164ea2be754d44c723c613be46` based on fresh download I did just now

Reply:
doh....